pyo3-ffi/src/methodobject.rs

```diff
@@ -94,6 +94,19 @@ pub struct PyMethodDef {
     pub ml_doc: *const c_char,
 }
 
+impl PyMethodDef {
+    pub const fn zeroed() -> PyMethodDef {
```

Comment:
I found it quite annoying to make a terminating PyMethodDef in a static, this would make it easier. I'm all for a better idea :)

Reply:
Yeah seems ok to me, it would be nice if `std::mem::zeroed` was `const`, but here we are 🙂